src/main/java/com/xceptance/xlt/engine/XltHttpWebConnection.java

```diff
@@ -486,4 +486,17 @@ protected static void putAdditionalRequestData(RequestData requestData, WebReque
             }
         }
     }
+
+    protected static String removeUserInfoIfNecessary(final URL url)
+    {
+        String urlString = url.toExternalForm();
+
+        // remove user-info from request URL if we need to (GH #57)
+        if(SessionImpl.REMOVE_USERINFO_FROM_REQUEST_URL)
+        {
+            urlString = UrlUtils.removeUserInfo(urlString);
+        }
+
+        return urlString;
+    }
 }
```

Comment:
In the result browser, the user info is still present in the URLs. Maybe we can fix this by removing the user info in `XltHttpWebConnection.cloneWebRequest()`?

Reply:
Done.

However, I'm not absolutely sure that this is a good idea as the result browser is primarily used for debugging and analysis purposes and users might wonder why this information is removed from there as well. As for me, I'd expect the result browser to represent the data that actually transmitted.

Reply:
Result browsers are often linked to in load test reports. If these reports are shared carelessly with other people, they might see this info again. So better safe than sorry.